### ISSUE TITLE
where is the loss function?

### ISSUE THREAD

**Issue by caizh666** [thtrieu/darkflow#376]
Where is the loss function? And, are those limitations resolved?

**Comment by simo23** [comment]
I think the loss function can be found in the train.py script. For example, for yolov2 it's here: https://github.com/thtrieu/darkflow/blob/master/darkflow/net/yolov2/train.py
Sorry but I don't know if the limitations have been resolved.

**Comment by qunsorber** [comment]
how to set the epoches number and batch size @simo23

**Comment by simo23** [comment]
I think you can change them in the .cfg file that you are using with batch_size and max_batches. Like in  darkflow/cfg/tiny-yolo-voc.cfg you can find these fields.
